app/models/post.rb

```diff
@@ -1,3 +1,7 @@
 class Post < ApplicationRecord
-  belongs_to :author_id
+  belongs_to :user
+
+  scope :last_user_posts, ->(user_id) { where(user_id:).last(3) }
```

Comment:
- [ ] I think this method it should be is user model as required  check my comment above 🙏🏻💯💯
![image](https://user-images.githubusercontent.com/58553711/220618206-1b91ef95-a23d-41f3-b6aa-fd59b89a04b6.png)